compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -1307,6 +1307,33 @@ class MulTestGraph final : public SimpleTestGraph
   luci::CircleConst *_const = nullptr;
 };
 
+template <Type T> class IntMulTestGraph final : public TypedTestGraph
+{
+public:
+  void init(void) override
+  {
+    TypedTestGraph::init(T, {32}, {32});
+
+    _const = create_dummy_const<T>(g(), {32});
+    _add = g()->nodes()->create<luci::CircleMul>();
```

Comment:
Maybe this is a typo.
```suggestion
    _mul = g()->nodes()->create<luci::CircleMul>();
```